pkg/bwagreement/server.go

```diff
@@ -42,6 +42,17 @@ type UplinkStat struct {
 	TotalTransactions int
 }
 
+//SavedOrder allows use of dbx.Bwagreement without the namespace
+type SavedOrder struct {
```

Comment:
You could also use `pb.Order`, although I do prefer database logic not be dependent on transport concerns.